src/Symfony/Component/Workflow/SupportStrategy/SupportStrategyInterface.php

```diff
@@ -11,10 +11,14 @@
 
 namespace Symfony\Component\Workflow\SupportStrategy;
 
+@trigger_error(sprintf('"%s" is deprecated since Symfony 4.1. Use "%s" instead.', SupportStrategyInterface::class, WorkflowSupportStrategyInterface::class), E_USER_DEPRECATED);
```

Comment:
I believe triggering here should be avoided as it's used internally, DebugClassLoader takes care of triggering if implemented in userland (@nicolas-grekas please correct me if I'm wrong)